R/uvhydrograph-data.R

```diff
@@ -22,7 +22,11 @@ parsePrimaryUVData <- function(data, month, useDownsampled=FALSE) {
   est_UV <- subsetByMonth(getTimeSeries(data, timeSeriesNames$primarySeriesName, estimatedOnly=TRUE), month)
   uncorr_UV <- subsetByMonth(getTimeSeries(data, timeSeriesNames$primarySeriesRawName ), month)
   comp_UV <- subsetByMonth(getTimeSeries(data, timeSeriesNames$comparisonSeriesName ), month)
-  water_qual <- subsetByMonth(getWaterQualityMeasurements(data), month)
+  water_qual <- tryCatch({
+    subsetByMonth(readWaterQualityMeasurements(data), month)
```

Comment:
The one liner is ok if errors from either function would be handled the same way. They only really need to be split if we have different responses to different error conditions.